diff --git a/node_modules/@solana/mpp/dist/server/Charge.js b/node_modules/@solana/mpp/dist/server/Charge.js
index 17b90ef..c3a9811 100644
--- a/node_modules/@solana/mpp/dist/server/Charge.js
+++ b/node_modules/@solana/mpp/dist/server/Charge.js
@@ -979,7 +979,7 @@ async function broadcastTransaction(rpcUrl, base64Tx) {
             id: 1,
             jsonrpc: '2.0',
             method: 'sendTransaction',
-            params: [base64Tx, { encoding: 'base64', skipPreflight: false }],
+            params: [base64Tx, { encoding: 'base64', preflightCommitment: 'confirmed', skipPreflight: false }],
         }),
         headers: { 'Content-Type': 'application/json' },
         method: 'POST',
